### PR TITLE
Completed credential signature data model

### PR DESCRIPTION
**Pull request by aritroCoder**
This pull request should be merged only after merging #162, otherwise the changes does not make much sense

### CODE REVIEW

File: spec/data_flow_issuance.md [hunk 1]
```diff
@@ -445,20 +445,13 @@ community.
...
-* `e` is the *TO BE ADDED*. It is constructed as follows:
-  * *TO BE ADDED*
-* `v` is the *TO BE ADDED*. It is constructed as follows:
-  * *TO BE ADDED*
+* `m_2` is a linkable identifier to the holder that is also called the `master_secret` in old versions. It is constructed as follows:
+  * $m_2 = H(i || \mathcal{H})$, where $i$ is an index assigned to the holder, and $\mathcal{H}$ is an identifier with which the [[ref: holder]] is known to the [[ref: issuer]].
+* `a` is the signature of the blinded known attributes. It's generation is given above.
+* `e` is a random prime generated by the [[ref: issuer]] for creating signature.
+* `v` is a number generated by the [[ref: holder]] to unblind the signature of the blinded attributes. It is constructed as follows:
+  * $v = v' + v''$, where $v'$ is the blinding factor which the holder has and $v''$ is the random number generated by the issuer.
```
Comment (swcurran) [inline spec/data_flow_issuance.md:454]: "and $v''$ is the random number generated by the issuer"

I think it should be a "is a random number...", as I assume they generate it on the fly?  I would think that the attributes (e.g. the size) of the number should be indicated?

**Review by mikelodder7 — APPROVED** [review]

File: spec/data_flow_issuance.md [hunk 2]
```diff
@@ -445,20 +445,13 @@ community.
...
-  * *TO BE ADDED*
-* `a` is the *TO BE ADDED*. It is constructed as follows:
-  * *TO BE ADDED*
-* `e` is the *TO BE ADDED*. It is constructed as follows:
-  * *TO BE ADDED*
-* `v` is the *TO BE ADDED*. It is constructed as follows:
-  * *TO BE ADDED*
+* `m_2` is a linkable identifier to the holder that is also called the `master_secret` in old versions. It is constructed as follows:
+  * $m_2 = H(i || \mathcal{H})$, where $i$ is an index assigned to the holder, and $\mathcal{H}$ is an identifier with which the [[ref: holder]] is known to the [[ref: issuer]].
+* `a` is the signature of the blinded known attributes. It's generation is given above.
```
Comment (mikelodder7) [inline spec/data_flow_issuance.md:451]: Add how is encoded. I believe for anoncreds 1.0 its base 10

Comment (aritroCoder) [reply spec/data_flow_issuance.md:451]: yes, it is encoded in decimal

Comment (aritroCoder) [reply spec/data_flow_issuance.md:444]: added this into the spec

**Review by swcurran — APPROVED** [review]

**Pull request merged**